### ISSUE TITLE
Change visibility of vertical tabs on Procedures

### ISSUE THREAD

**Issue by padi18** [dbeaver/dbeaver#36266]
Hello! This is my first issue here on GitHub. I’m from Spain, so sorry for my English!

I want to suggest a feature that I think is not very difficult and has many benefits. It is a visual feature:

![image](https://github.com/user-attachments/assets/f53d2a48-4b0d-4e7d-aa73-9ca47aba2263)

The tabs here are vertical, but why not make them horizontal like the top tabs for properties, data, and schema? This way, we would have more visibility for our procedure source! And not only for procedures, but for all sections with vertical tab formats!

Thanks!

**Comment by E1izabeth** [comment]
Thank you for the feature request.

**Comment by ask9** [comment]
> we would have more visibility for our procedure source

I also agree that this part of UI takes a lot of space decreasing the area for procedure sources. 
I would suggest showing only buttons without text (text is shown on mouse hover over the button) or add another button to toggle between showing/hiding them.
See the gif below:

![Animation](https://github.com/user-attachments/assets/f82d0a7b-3cb0-4a25-8a29-3ebd90928f4d)
